ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -401,6 +401,7 @@ SFVSC:
   - supply_fan_failed_alarm
   - control_mode
   - supply_fan_power_status
+  - master_alarm
```

Comment:
Two comments:
1. Is this alarm name appropriate? Is there a more specific name that can be used.
2. This is not appropriate for the VFD abstract type. Why is this not defined on the canonical type?

Reply:
1. Here in our context the alarm is an master alarm for example if  any alarm is active then the master alarm will be active.
2. Will add that as part of canonical type.